lib/abbreviato/truncated_sax_document.rb

```diff
@@ -0,0 +1,194 @@
+require 'nokogiri'
+require 'htmlentities'
+
+class TruncatedSaxDocument < Nokogiri::XML::SAX::Document
+  IGNORABLE_TAGS = %w[html head body].freeze
```

Comment:
At the moment, these elements get stripped. Depending on the nature of the HTML documents/fragments we're going to be truncating, this part will need to be re-visited. Do we know if the email parts will still have these elements by the time they need to be truncated? @alduethadyn @rmartz00